backend/backend/backend-src/docs/api/events/get-event-by-address.ts

```diff
@@ -0,0 +1,54 @@
+export default module.exports = {
+  get: {
+    tags: ['Events'],
+    parameters: [
+      {
+        in: 'path',
+        name: 'address',
+        required: true,
+        schema: {
+          type: 'string',
+          example: '0x7b176b89ab5ed899d17b05ffb67b39eeda8aca3e7f41e40353937ed8c943725e',
+        },
+      },
+    ],
+    responses: {
+      '200': {
+        description: 'Successful response',
+        content: {
+          'application/json': {
+            schema: {
+              type: 'object',
+              properties: {
+                status: {type: 'string', example: 'ok'},
+                events: {
+                  type: 'array',
+                  items: {
+                    type: 'object',
+                    properties: {
+                      // Define properties for events
```

Comment:
@Imamah-Zafar  
I think something is missing here.

Reply:
This endpoint needs to distinguish between events stored against user addresses and events stored against package addresses.  @asimm241

Reply:
in the recent commit i have updated the URL so we have 
```/v2/events/package/:address/:ticker``` and ```'/v2/events/address/:address``` endpoints to fetch package event and address event respectively